WooCommerce/src/main/kotlin/com/woocommerce/android/ui/coupons/details/CouponDetailsScreen.kt

```diff
@@ -59,10 +66,10 @@ fun CouponDetailsScreen(
                     expanded = showMenu,
                     onDismissRequest = { showMenu = false }
                 ) {
-                    DropdownMenuItem(onClick = { /*TODO*/ }) {
+                    DropdownMenuItem(onClick = { onCopyButtonClick() }) {
```

Comment:
Just sharing a tip, you don't need to make any changes.

The version below is better optimized than the one in the code:
```suggestion
                    DropdownMenuItem(onClick = onCopyButtonClick()) {
```

This is because as Android pre-API26 doesn't support lambdas, so to support them, Kotlin generates a class that implements a special interface (for this case it's [Function0](https://github.com/JetBrains/kotlin/blob/master/spec-docs/function-types.md#functions-with-022-parameters-at-runtime)), then it instantiates this interface when we invoke the lambda.

So when assigning the reference `onCopyButtonClick` to `onClick` directly, we avoid this.

Reply:
> Kotlin generates a class that implements a special interface

TIL! Thanks for that interesting fact.

Reply:
I tried the above and got this error 🤔 

<img width="781" alt="Screen Shot 2022-04-19 at 04 39 24" src="https://user-images.githubusercontent.com/266376/163882182-e417fb3f-d0d3-4db3-8f0c-f6721ea0b960.png">

Reply:
Sorry for this, the trailing parentheses are not needed, I fixed the suggestion.